boards/arm/pico_pi_m4/pico_pi_m4.dts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018, Joris Offouga
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+#include <nxp/nxp_imx7d_m4.dtsi>
+/ {
+	model = "Pico-Pi IMX7D board";
+	compatible = "nxp,mcimx7d_m4";
+
+	aliases {
+
+		uart-6 = &uart6;
+	};
+
+	chosen {
+#if defined(CONFIG_XIP)
```

Comment:
Is this ifdef needed?

Reply:
No it's not required, i remove it

Reply:
@galak,

This is related to the "dummy-flash" thing.